### PR TITLE
Bug 1491726 - remove old IP blocks for signing

### PR DESCRIPTION
**Pull request by nthomas-mozilla**
I did a bunch of grepping to make sure nothing is using the removed definitions.

### CODE REVIEW

File: modules/fw/manifests/networks.pp [hunk 1]
```diff
@@ -36,18 +36,10 @@
     # AWS Networks; See https://github.com/mozilla-releng/build-cloud-tools/blob/master/configs/subnets.yml
 
     # USE1 Network CIDRs
```
Comment (escapewindow) [inline modules/fw/manifests/networks.pp]: We may want to verify that these are buildbot-worker-only subnets... i.e., no taskcluster workers.

File: modules/fw/manifests/roles/dep_signing_from_osx.pp [hunk 2]
```diff
@@ -8,10 +8,6 @@
     fw::rules { 'allow_dep_signing_from_osx':
         # All signing sources
         sources =>  [   $::fw::networks::all_partner_repack,
```
Comment (escapewindow) [inline modules/fw/manifests/roles/dep_signing_from_osx.pp]: do we need `all_partner_repack` anymore? in this file and the other `*signing_from_*` files.

File: modules/fw/manifests/networks.pp (hunk 1 above)
Comment (dividehex) [reply modules/fw/manifests/networks.pp]: If these subnets are verified not be in use anymore, we should make sure to delete them from the vpc

File: modules/fw/manifests/networks.pp [hunk 3]
```diff
@@ -36,18 +36,10 @@
...
-    $use1_test    = [ '10.134.40.0/21', '10.134.56.0/22', '10.134.60.0/22', '10.134.156.0/22' ]
-    $use1_build   = [ '10.134.52.0/22', '10.134.164.0/24' ]
-    $use1_srv     = [ '10.134.48.0/22' ]
```
Comment (dividehex) [inline modules/fw/manifests/networks.pp]: srv has typically been were we put infra like puppet masters and log aggregators.  I assume it should stick around.

File: modules/fw/manifests/networks.pp (hunk 1 above)
Comment (nthomas-mozilla) [reply modules/fw/manifests/networks.pp]: aki, I've checked and they don't overlap. In use1, signing scriptworkers are all in 10.134.30.0/24, and non-signing scriptworkers in 10.134.48.0/24; in usw2, all scriptworkers are in 10.132.30.0/24.

dividehex, and update cloud-tools I guess. More than I was planning on right now, I'll see if I can get to it.

File: modules/fw/manifests/networks.pp (hunk 3 above)
Comment (nthomas-mozilla) [reply modules/fw/manifests/networks.pp]: Agreed, and we put non-signing scriptworkers in there too for use1, but this definition doesn't seem to be used anywhere else in puppet. Would you prefer to leave it as a reference ?

File: modules/fw/manifests/roles/dep_signing_from_osx.pp (hunk 2 above)
Comment (nthomas-mozilla) [reply modules/fw/manifests/roles/dep_signing_from_osx.pp]: Nope, I'll remove it.

File: modules/fw/manifests/networks.pp (hunk 3 above)
Comment (dividehex) [reply modules/fw/manifests/networks.pp]: Yes.  I'd prefer to keep the reference even if it isn't used.

**Pull request merged**